src/routes/console/project-[project]/auth/security/+page.svelte

```diff
@@ -81,6 +82,28 @@
             });
         }
     }
+    async function updateSessionsLimit() {
+        try {
+
+            let path = '/projects/' + projectId + '/auth/max-sessions';
```

Comment:
I think this can be a `const` too :)

Reply:
This will change once we release the SDK